src/daft-sql/src/lib.rs

```diff
@@ -305,9 +305,7 @@ mod tests {
                 Some(vec![null_equals_null]),
                 JoinType::Inner,
                 None,
-                None,
-                Some("tbl3."),
-                true,
+                JoinColumnRenamingParams::builder().prefix("tbl3.").build(),
```

Comment:
nit/suggestion: just a personal preference, but I think full builder pattern here is a bit overkill. 

IMO this would be a lot cleaner
```rs
JoinColumnRenamingParams::default().prefix("tbl3.")
```

Reply:
i think this is called the "fluent interface" pattern